.github/workflows/test.yml

```diff
@@ -0,0 +1,60 @@
+on:
+  pull_request:
+    branches:
+      - main
+      - master
+  push:
+    branches:
+      - main
+      - master
+  release:
+    types: [published]
+
+env:
+  GOFLAGS: "-mod=vendor"
+  GO111MODULE: on
+
+jobs:
+  test:
+    name: Test and Publish
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+
+      - uses: actions/setup-go@v5
+        with:
+          go-version: '1.16'
+
+      - name: Setup
+        run: |
+          make tools
+          go mod vendor
```

Comment:
`make tools` runs `go mod vendor` so I'm not sure why it's also run after in travis.

Reply:
:shrug: